util/regexp-assemble/data/932175.data

```diff
@@ -0,0 +1,27 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regexp_assemble/.
+
+##! Prevent command aliasing
+
+##! starting tokens prefix
+##!> assemble
+\b
+##!=>
+    ##!> cmdline unix
+alias
+    ##!<
+##!=>
+
+##! match possible white space between prefix expressions
+\b\s+
```

Comment:
According to the comment, the white space is optional, the expression makes it mandatory. ZSH at least enforces _no_ spaces:
```shell
# fails:
$ alias "foo" = "ls"
zsh: bad assignment
foo=''

# succeeds but lists a match
$ alias "foo"= "ls"
ls='ls -G'

# succeeds but lists a match
$ alias "foo" ="ls"
foo=''

# succeeds
$ alias "foo"="ls"
$ alias foo
foo=ls
```

Reply:
After testing with bash, yeah, no space should be the only that works.

Reply:
Wait. This is between the word alias and the name. We need at least one space.

Reply:
You're right, there has to be one between alias and the argument. The comment should say `match white space between command and arguments`.

Reply:
Done.